src/commons/DBReader.h

```diff
@@ -187,7 +187,7 @@ class DBReader : public MemoryTracker {
 
     size_t getSize() const;
 
-    unsigned int getProteomeTotalLen(size_t id); //gyuri
+    unsigned int getProteomeTotalLen(size_t id); 
```

Comment:
I would name this `getSourceTotalLen` or `getSetTotalLen` to be consistent with the `.source` naming